### DISCUSSION TITLE
Webhook per invio ad hub digitale

### DISCUSSION THREAD

**Discussion by FrancescoAntonelli** [fattureincloud/api#383] [Q&A] [answered]
Ciao,
  è possibile in qualche maniera intercettare l'evento di Firma e Invia?

Grazie
   Francesco

**Comment by emajo** [comment, marked as answer]
Ciao @FrancescoAntonelli, tra gli eventi registrabili abbiamo l'evento `it.fattureincloud.webhooks.issued_documents.e_invoices.status_update` che manderà un webhook ogni qualvolta lo stato del documento elettronico cambi.
![ei_status](https://github.com/fattureincloud/api/assets/48432054/8bffcb78-cb5e-468e-b631-81909f6ba3cf)
Per intercettare solamente l'invio del documento è necessario fare una chiamata [Get Issued Document](https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/issued_documents/-document_id-) all'arrivo del webhook e controllare che il valore del campo `ei_status` sia diventato `sent`. 

Grazie,
Emanuele.
